structuralcodes/materials/concrete/_concreteEC2_2023.py

```diff
@@ -237,22 +235,22 @@ def fctd(self, t_ref: float = 28) -> float:
 
     @property
     def eps_c1(self) -> float:
-        """Returns the strain at maximum compressive strength
-            of concrete (fcm) for the Sargin constitutive law.
+        """Returns the strain at maximum compressive strength of concrete (fcm)
+        for the Sargin constitutive law.
 
         Returns:
-            float: the strain at maximum compressive strength of concrete
+            float: The strain at maximum compressive strength of concrete.
         """
         self._eps_c1 = self._eps_c1 or ec2_2023.eps_c1(self.fcm)
         return self._eps_c1
 
     @eps_c1.setter
     def eps_c1(self, value: float):
-        """Sets a user defined value for strain at peak strenght
-        for Sargin constitutive law.
+        """Sets a user defined value for strain at peak strength for Sargin
+        constitutive law.
 
-        Args:
-            value (float): the new value for eps_c1, no units
+        Arguments:
+            value (float): The new value for eps_c1, no units.
```

Comment:
Just to be consistent.: previously (I think in creep or shrinkage) strain was defined in mm/mm. We should decise and name them consistently.

Reply:
Good point, I am fine with both variants, perhaps inclined towards mm/mm. But to be clear, the numbers shouldn't change even though we change from mm/mm to no units?

Reply:
Exactly the numbers are the same when given without units (as pure number) or in mm/mm (that actually cancel out).
The problem is when strains is given in % or in ‰ or even in more exotic units like mm/m (that actually should be a strange synonim of ‰).